### ISSUE TITLE
Logspam

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#4054]
The JUPnP trace logging is really excessive, I know you had a fix for that a while ago @SurfaceS and I asked you to revert it, but maybe we should bring it back? It makes it hard to use the logs for other things

**Comment by SurfaceS** [comment]
I think that UMS not enough use the info level (most of it is under debug), and some trace should be on the debug level.
And so, the default UMS log level should be info.
We should be able to debug from the debug level, and trace on few need if debug is not enough.

I'll have a look on it to shut up some JUPnP logging.

**Comment by SubJunk** [comment]
I agree we have used `trace` too much, a lot of it should be `debug`

I think the reason we didn't use `info` much is because some users like looking at the Logs tab which defaults to `info`, so we tried to keep that clean

**Comment by SurfaceS** [comment]
For this, I will need to know what you want to keep from it.

**Comment by SubJunk** [comment]
v14 looks good for this
